src/components/InvoiceTitle/index.jsx

```diff
@@ -0,0 +1,11 @@
+import { Link } from "react-router-dom";
+
+export default function InvoiceTitle({invoiceID}) {
+
+    return (
+        <div className="flex justify-between border-b border-slate-500">
+            <p className="ml-9">Invoice <span className=" text-slate-500">#</span>{invoiceID}</p>
```

Comment:
Can you make this text bold and reduce the margin left to better match the designs